src/static/css/content.less

```diff
@@ -288,20 +289,25 @@ figure {
 }
 /* Landscape tablet/netbook/laptop (1024px) */
 @media only screen and (min-width: 64em) {
-    .swatch {
+    .swatch__secondary {
         .grid_column(3);
     }
+
+    .swatch__primary {
+        .grid_column(6);
+    }
 }
 
-@colors: 'green', 'green-midtone', 'green-tint', 'black', 'darkgray',
-         'gray', 'gray-80', 'gray-50', 'gray-20', 'gray-10', 'gray-5',
-         'redorange', 'redorange-80', 'redorange-50', 'redorange-20',
-         'gold', 'gold-80', 'gold-50', 'gold-20',
-         'neutral', 'neutral-80', 'neutral-50', 'neutral-20',
-         'navy', 'navy-80', 'navy-50', 'navy-20',
-         'pacific', 'pacific-80', 'pacific-50', 'pacific-20',
-         'teal', 'teal-80', 'teal-50', 'teal-20',
-         'dark-redorange';
+@colors: 'dark-green', 'green', 'green-80', 'green-60', 'green-40', 'green-20',
+         'dark-teal', 'teal', 'teal-80', 'teal-60', 'teal-40', 'teal-20',
+         'dark-pacific', 'pacific', 'pacific-80', 'pacific-60', 'pacific-40', 'pacific-20',
+         'dark-navy', 'navy', 'navy-80', 'navy-60', 'navy-40', 'navy-20',
+         'dark-purple', 'purple', 'purple-80', 'purple-60', 'purple-40', 'purple-20',
+         'dark-red', 'red', 'red-80', 'red-60', 'red-40', 'red-20',
+         'dark-gold', 'gold', 'gold-80', 'gold-60', 'gold-40', 'gold-20',
+         'dark-neutral', 'neutral', 'neutral-80', 'neutral-60', 'neutral-40', 'neutral-20',
+         'black', 'dark-gray', 'gray', 'gray-80', 'gray-60', 'gray-40', 'gray-20',
+         'gray-10', 'gray-5', 'white';
```

Comment:
Yeah, this is on my "to fix" list but in another PR. This is already a bit more complex than I'd like.